### PR TITLE
Upgraded redux-saga related dependencies and used babel-preset-env.

### PR DESCRIPTION
**Pull request by visusnet**
This PR updates history, react, react-dom, react-redux, react-router, redux-saga (incl. peer dependency), redux-saga-test-plan and replaced babel-preset-es2015 by babel-preset-env.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,18 +63,18 @@
...
-    "redux-saga-test-plan": "^2.4.4",
+    "redux-saga": "^0.16.0",
+    "redux-saga-test-plan": "^3.4.0",
     "rimraf": "^2.5.4",
     "webpack": "^1.13.2",
     "webpack-dev-server": "^1.16.1"
   },
   "peerDependencies": {
-    "redux-saga": ">=0.10.0 <0.16.0"
+    "redux-saga": ">=0.10.0 <=0.16.0"
```
Comment (jfairbank) [inline package.json]: Should this be `>=0.10.0 <0.17.0` instead to include patch releases of `0.16`?

**Review by jfairbank — CHANGES_REQUESTED** [review]
Thanks for the PR @visusnet! One minor thing and we're good to go.

**Comment by visusnet** [comment]
@jfairbank You are right. I‘ve updated the PR accordingly.

**Review by jfairbank — APPROVED** [review]

**Comment by jfairbank** [comment]
Awesome, thanks!

**Pull request merged**